src/identifier.h

```diff
@@ -21,12 +21,18 @@
 
 class Identifier : public RootObject
 {
+private:
```

Comment:
> GDC has a few dozen examples of using ident->string directly in the code generator

That suggests there's a member function lacking. What would that be? I've purged all these direct accesses from dmd. (How is GDC bypassing the `private:` anyway?)

> Having these fields present is also a win for gdb

That is a better point, but it still seems unnecessary.

The larger point is that we should be working towards encapsulating data with interfaces, not poking about in their innards.